source/_docs/guides/front-end-performance.md

```diff
@@ -0,0 +1,457 @@
+
+<dt>age</dt>
+<dd markdown="1">The number of seconds cache has been available to serve the request. Any number greater than zero indicates that this response was served to the browser from cache.</dd>
+<dt>cache-control</dt>
+<dd markdown="1">This header should include a `max-age` that is the maximum number of seconds that the cache can be kept.</dd>
+<dt>surrogate-key-raw</dt>
+<dd>Metadata including the the content IDs for what was displayed on this page. This metadata instructs this page to be cleared from cache when any of those posts are saved again.</dd>
+<dt>x-served-by</dt>
+<dd>This header indicates which POP your response came from. Our primary infrastructure is in the Midwest of the United States so the first item you will probably see on this list will include "ORD" for the O'Hare airport in Chicago. If you're physically located in Austin you will also see DFW, indicating the response went from the primary datacenter to a cached copy in Chicago to a cached copy in Dallas.</dd>
+
+![Chrome network headers](/source/docs/assets/images/guides/front-end-performance/chrome-network-headers.png)
+
+<div class="panel panel-drop panel-guide" id="accordion">
+  <div class="panel-heading panel-drop-heading">
+    <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#dev-tools">
+      <h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-lightbulb"></span> Google's Developer Tools</h3>
+    </a>
+  </div>
+  <div id="dev-tools" class="collapse" markdown="1" style="padding:10px;">
+  Examine the HTTP headers for the response using Chrome's Developer tools:
+
+  ![Check headers in chrome](/source/docs/assets/images/guides/front-end-performance/inspect-network.gif)
+
+  1. Right click somewhere on the page.
+  2. Select **Inspect**.
+  3. Open the **Network** tab, then refresh the page. This allows you to review all of the HTTP requests necessary to display your page.
+  4. Select the first row, which is a request for the initial HTML file.
+  5. Look for the **Age** header. Any number greater than zero indicates that this response was served to the browser from cache.
+  </div>
+</div>
+
+### Debug Common Cache Busters
+#### Cookies
+Cookies are included in the response headers we examined previously. They can include sessions for authenticated traffic to logged in users, which can invalidate cache. For WordPress, it's common for plugins to add their own cookies in such a way that breaks full-page caching.
+
```

Comment:
We should probably list cookies that can break caching here, plus how to see if your site is sending them unnecessarily?

Reply:
@stevector @joshkoenig I added the cookie patters, but I need some help filling in the content on how to see if your site is sending them unnecessarily.

Reply:
I'm adding instructions

Reply:
https://github.com/pantheon-systems/documentation/pull/2896/commits/9c02dc26e26e770fb26cffb4fb75a4620cd84658